src/main/java/org/sagebionetworks/bridge/spring/controllers/AdherenceController.java

```diff
@@ -91,4 +94,28 @@ public StatusMessage updateAdherenceRecords(@PathVariable String studyId, @PathV
         
         return service.getAdherenceRecords(session.getAppId(), search);
     }
+
+    @DeleteMapping("/v5/studies/{studyId}/participants/{userIdToken}/adherence/{instanceGuid}/{eventTimestamp}/{startedOn}")
+    public StatusMessage deleteAdherenceRecord(
+            @PathVariable String studyId,
+            @PathVariable String userIdToken,
+            @PathVariable String instanceGuid,
+            @PathVariable String eventTimestamp,
+            @PathVariable String startedOn) {
+        UserSession session = getAuthenticatedSession(RESEARCHER, STUDY_COORDINATOR);
+
+        String userId = accountService.getAccountId(session.getAppId(), userIdToken)
+                .orElseThrow(() -> new EntityNotFoundException(Account.class));
+
+        AdherenceRecord record = new AdherenceRecord();
+        record.setAppId(session.getAppId());
+        record.setUserId(userId);
+        record.setStudyId(studyId);
+        record.setInstanceGuid(instanceGuid);
+        record.setInstanceTimestamp(DateTime.parse(startedOn));
+        record.setEventTimestamp(DateTime.parse(eventTimestamp));
```

Comment:
Consider using BridgeUtils.getDateTimeOrDefault(eventTimestamp, null) for the timestamps. This will set these values to null, and then you should validate they are set in the service. Otherwise, the user with get a cryptic error related to DateTime parsing that doesn't explicitly tell them what is wrong.

Reply:
I switched to BridgeUtils.getDateTimeOrDefault() but I just realized I haven't validated those fields in the service. I'll make a quick update and add unit tests for it.